sdk/tests/js/tests/tex-image-and-sub-image-2d-with-image-bitmap.js

```diff
@@ -89,7 +89,7 @@ function generateTest(internalFormat, pixelFormat, pixelType, prologue, resource
         gl.texParameteri(bindingTarget, gl.TEXTURE_MAG_FILTER, gl.NEAREST);
         // Set up pixel store parameters
         gl.pixelStorei(gl.UNPACK_FLIP_Y_WEBGL, flipY);
-        gl.pixelStorei(gl.UNPACK_PREMULTIPLY_ALPHA_WEBGL, premultiplyAlpha);
+        gl.pixelStorei(gl.UNPACK_PREMULTIPLY_ALPHA_WEBGL, false);
```

Comment:
I disagree with such fix.  It silently changes the test parameters.

Instead, we basically should change the caller side and make sure we don't call runOneIteration(premultuplyAlpha==true), and add an assert in runOneIteration.

Or simply remove the premultiplyAlpha arg and we don't even need to call pixelStorei as its default is false.